src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/TextBox.cs

```diff
@@ -463,15 +463,15 @@ public  bool CanUndo
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.SelectionChanged.remove
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.ContextMenuOpening.add
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.ContextMenuOpening.remove
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
+		#if false || false || NET461 || false || false || __NETSTD_REFERENCE__ || false
+		[global::Uno.NotImplemented("NET461", "__NETSTD_REFERENCE__")]
 		public  void Select( int start,  int length)
```

Comment:
What does it take to implement this for NET461 and .NET Standard (I'm not sure what does it mean to implement it for these since they're just target frameworks, not platforms. ie, it's already implemented for android, ios, mac, skia, wasm. So what's left?)

Reply:
`NET461` methods can generally be left as empty stubs, to be filled in later if ever necessary for a specific unit test